src/System.Windows.Forms/src/System/Windows/Forms/Controls/PropertyGrid/PropertyGridInternal/GridEntry.cs

```diff
@@ -321,7 +333,9 @@ internal virtual bool InternalExpanded
             // Accessible clients won't see this unless both events are raised.
 
             // Root item is hidden and should not raise events
-            if (OwnerGridView.IsAccessibilityObjectCreated && GridItemType != GridItemType.Root)
+            if (OwnerGridView is not null
+                && OwnerGridView.IsAccessibilityObjectCreated
```

Comment:
```suggestion
            if (OwnerGridView is { } owner
                && owner.IsAccessibilityObjectCreated
```
same can be done below as well